src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,32 @@
+import { TfiClose } from 'react-icons/tfi';
+
+type PropTypes = {
+  showModal: boolean;
+  onClose: () => void;
+  children: React.ReactNode;
```

Comment:
I want you to try something new. I want you to apply `PropsWithChildren` type and remove `children` https://ultimatecourses.com/blog/use-intersection-types-to-combine-types-in-typescript